keps/sig-instrumentation/20190605-metrics-validation-and-verification.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Metrics Validation and Verification
+authors:
+  - "@serathius"
+  - "@solodov"
+  - "@logicalhan"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-instrumentation
+  - sig-testing
```

Comment:
Who from sig testing is participating in this? Has this been reviewed by sig-testing at all?
cc: @brancz @serathius @solodov @logicalhan